### PR TITLE
Fix pycodestyle E741 and re-enable checks

### PR DESCRIPTION
**Pull request by jmacarthur**

### CODE REVIEW

**Comment by jmacarthur** [comment]
Fixes #260.

File: .pycodestyle.ini [hunk 1]
```diff
@@ -6,4 +6,4 @@
 # E711: comparison to None (used to improve test style)
 # E712: comparison to True (used to improve test style)
 # E741 ambiguous variable name
```
Comment (jacebrowning) [inline .pycodestyle.ini]: You can remove this line too.

**Comment by jmacarthur** [comment]
Good point! Fixed!

**Comment by jacebrowning** [comment]
I'm going to pin the version of `pydocstyle` in `develop`. The new release has additional checks were not yet using.

**Pull request merged**